app/javascript/bundles/RestaurantsApp/components/AddRestaurant.jsx

```diff
@@ -62,6 +65,7 @@ export default class AddRestaurantForm extends React.Component {
             };
 
             const data = JSON.stringify(restaurant);
```

Comment:
`fetch` might be able to handle non-stringified data (but I'm not certain).

Reply:
Looks like it needs the stringify